### PR TITLE
Cache CIDR blocks for whitelists and blacklists to speed response time 

### PR DESCRIPTION
**Pull request by davidhanley**

### CODE REVIEW

**Comment by abedra** [comment]
Thanks! The real diff is a little hard to follow with all the commits, but this looks like an overall win. I'm going to spend a little longer looking things over but I'll likely merge it this weekend and clean anything up that got missed separately.

**Comment by davidhanley** [comment]
I think you should be able to look at the result of all the commits... If
not, i'll squash them together.  Sorry for all those, i like to make a
little tiny change, test, commit--sometimes to an absurd degree.

On Fri, Aug 28, 2015 at 4:56 PM, Aaron Bedra <notifications@github.com>
wrote:

> Thanks! The real diff is a little hard to follow with all the commits, but
> this looks like an overall win. I'm going to spend a little longer looking
> things over but I'll likely merge it this weekend and clean anything up
> that got missed separately.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/repsheet/librepsheet/pull/30#issuecomment-135897605>.
>



-- 
"When I let go of what I am, I become what I might be."
—Lao Tzu
